### PR TITLE
Add basic CaSR notebook

### PR DESCRIPTION
**Pull request by huard**
Based on existing nb from @tlogan2000

### CODE REVIEW

**Comment by tlogan2000** [comment]
@huard Do you want me to quickly create the renamed 'CaSR' ncml for this notebook?

**Comment by huard** [comment]
I don't think there is a rush to change the name. My understanding is that the critical point is getting a URL for a notebook. So I suggest we get a first version out the door and update the name later on.

File: docs/source/notebooks/CaSR_basic.ipynb [hunk 1]
```diff
@@ -0,0 +1,6506 @@
+{
```
Comment (tlogan2000) [inline docs/source/notebooks/CaSR_basic.ipynb:1]: <p>One small suggestion :  As the link to this notebook will be provided on CaSPAR etc I think we should show that there is a <code>daily</code> and <code>hourly</code> ncml directly here</p><p> </p><p>e.g. open and display both.  Then use the daily in the rest of the notebook  </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/pavics-sdi/pull/320/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (tlogan2000) [reply docs/source/notebooks/CaSR_basic.ipynb:1]: <p>or at least give the url for hourly even if the variable is never used?  <code>url_1hr="https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/dodsC/datasets/reanalyses/1hr_RDRSv2.1_NAM.ncml"</code>  </p>

Comment (huard) [reply docs/source/notebooks/CaSR_basic.ipynb:1]: <p>It's in the text just above Data Access</p>

**Review by tlogan2000 — APPROVED** [review]

**Pull request merged**